SUPPORTEDTECH.rst

```diff
@@ -0,0 +1,111 @@
+Supported Technology
+====================
+
+Supported Operations
+####################
+
+.. table::
+
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | **Protocol** | **Security Guarantee** | **Operation**                           | **Number of Parties** |
+    +==============+========================+=========================================+=======================+
+    | ABY3         | Semi-honest            | Addition                                |                    2+ |
```

Comment:
Also, don't add ABY3 in supported ops. Just FSS, SPDZ, and Falcon. ABY3 just provides truncation and in the future will provide share conversion protocols.